master/docs/manual/customization.rst

```diff
@@ -932,18 +932,72 @@ For more information on the available commands, see :doc:`../developer/master-wo
     Step Progress
     BuildStepFailed
 
-Writing New Status Plugins
---------------------------
+Writing Dashboards with Flask
+-----------------------------
+
+Buildbot Nine UI is written in Javascript.
+This allows it to be reactive and realtime, but comes at a price of a fair complexity.
+Sometimes, you just need a dashboard displaying your build results in your own weird manner but learning angularJS for that is just too much.
+
+There is a buildbot plugin, which allows to write a server side generated dashboard, and integrated it in the UI.
+
+.. code-block:: python
 
-Each status plugin is an object which provides the :class:`twisted.application.service.IService` interface, which creates a tree of Services with the buildmaster at the top [not strictly true].
```

Comment:
Is this text outdated or present somewhere else? I don't see it being moved elsewhere.